imperial_coldfront_plugin/migrations/0011_auto_20250401_1022.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.11 on 2025-04-01 10:22
+
+from django.db import migrations
+import datetime
+from django_q.tasks import schedule
+
+
+def run_consistency_check(apps, schema_editor):
+    """Schedule a task to run the consistency check."""
+    Schedule = apps.get_model("django_q", "Schedule")
+    schedule(
+        "imperial_coldfront_plugin.tasks.check_allocation_user_consistency",
+        schedule_type="I",
+        minutes=60,
+        repeats=-1,
+        next_run=datetime.datetime.now(),
+    )
```

Comment:
Not checked the below but you need to use the Schedule model.
```suggestion
    Schedule.objects.update_or_create(
        name="LDAP consistency check",
        func="imperial_coldfront_plugin.tasks.check_allocation_user_consistency",
        schedule_type="D",
    )
```